lib/message_bus/rack/middleware.rb

```diff
@@ -54,7 +55,7 @@ def stop_listener
   # Process an HTTP request from a subscriber client
   # @param [Rack::Request::Env] env the request environment
   def call(env)
-    return @app.call(env) unless env['PATH_INFO'] =~ /^\/message-bus\//
+    return @app.call(env) unless env['PATH_INFO'].start_with? "#{@bus.base_route}message-bus/"
```

Comment:
I know I am being ultra stickler here, but might as well micro optimise this cause it is called on every request.

Maybe memoize this value at `@bus.base_path` then you just keep one of these strings around for the life of the process.